orca-clouddriver/src/test/java/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/WaitForManifestStableTaskTest.java

```diff
@@ -273,6 +275,44 @@ void waitsForAllManifestsWhenOneFailedAndOneUnknown() {
         .containsExactly(failedMessage(MANIFEST_1), waitingToStabilizeMessage(MANIFEST_2));
   }
 
+  @Test
+  void waitTaskContextIsRestartedWhenClouddriverReturnsException() {
+    OortService oortService = mock(OortService.class);
+    WaitForManifestStableTask task = new WaitForManifestStableTask(oortService);
+
+    StageExecutionImpl myStage =
+        createStageWithManifests(
+            ImmutableMap.of(NAMESPACE, ImmutableList.of(MANIFEST_1, MANIFEST_2)));
+
+    when(oortService.getManifest(ACCOUNT, NAMESPACE, MANIFEST_1, false))
+        .thenReturn(manifestBuilder().stable(true).failed(false).build());
+    when(oortService.getManifest(ACCOUNT, NAMESPACE, MANIFEST_2, false))
+        .thenReturn(manifestBuilder().stable(false).failed(false).build());
+
+    TaskResult result = task.execute(myStage);
+    AssertionsForClassTypes.assertThat(result.getStatus()).isEqualTo(ExecutionStatus.RUNNING);
+    assertThat(getMessages(result)).containsExactly(waitingToStabilizeMessage(MANIFEST_2));
+    assertThat(getErrors(result)).isEmpty();
+
+    reset(oortService);
+
+    when(oortService.getManifest(ACCOUNT, NAMESPACE, MANIFEST_2, false))
+        .thenThrow(
+            new SpinnakerServerException(new Request.Builder().url("http://localhost").build()));
+
+    result =
+        task.execute(
+            createStageWithContext(
+                ImmutableMap.<String, Object>builder()
+                    .putAll(myStage.getContext())
+                    .putAll(result.getContext())
+                    .build()));
+
+    AssertionsForClassTypes.assertThat(result.getStatus()).isEqualTo(ExecutionStatus.RUNNING);
```

Comment:
please use (it's already statically imported)`org.assertj.core.api.Assertions.assertThat`.
```suggestion
    assertThat(result.getStatus()).isEqualTo(ExecutionStatus.RUNNING);
```